ee/src/main/resources/org/jboss/as/ee/subsystem/LocalDescriptions.properties

```diff
@@ -55,6 +55,7 @@ managed-executor-service.hung-thread-count=The number of executor threads that a
 managed-executor-service.max-thread-count=The largest number of executor threads.
 managed-executor-service.task-count=The approximate total number of tasks that have ever been submitted for execution.
 managed-executor-service.thread-count=The current number of executor threads.
+managed-executor-service.refuse-lifecycle-operation=Lifecycle operation not supported.
```

Comment:
I don't think this should be here.  This file provides the English language version of free-form text strings used in the WildFly management API description (i.e. what you see if you do /:read-resource-description or read-operation-description in the CLI). It's not used for anything else.

Reply:
Thank you. I have reverted the change to LocalDescriptions.properties.